src/main/java/uk/gov/hmcts/divorce/divorcecase/tab/ApplicationTab.java

```diff
@@ -328,7 +330,10 @@ private void addApplicant2Representation(final Tab.TabBuilder<CaseData, UserRole
             .field("applicant2SolicitorPhone", "applicant2SolicitorRepresented!=\"No\"")
             .field("applicant2SolicitorEmail", "applicant2SolicitorRepresented!=\"No\"")
             .field("applicant2SolicitorFirmName", "applicant2SolicitorRepresented!=\"No\"")
-            .field("applicant2SolicitorOrganisationPolicy", "applicant2SolicitorRepresented!=\"No\"")
+            .field(
+                "applicant2SolicitorOrganisationPolicy",
+                "applicant2SolicitorRepresented!=\"No\" AND applicant2SolicitorOrganisationPolicy.Organisation.OrganisationID=\"*\""
+            )
```

Comment:
I looked through the commit/ticket history and found there was a reason behind the applicant 2 show condition being changed to use `"applicant2SolicitorRepresented!=\"No\""`:
https://github.com/hmcts/nfdiv-case-api/pull/1928/files#r881473119

To avoid introducing bugs, I decided on minimal adjustments in this ticket - just enough to ensure we don't start showing the blank org policies that are being set in preparation for NoC. EXUI seems to use the OrgID to retrieve the organisation details and show them in EXUI and meaning that when this is not set, the organisation will appear blank and it makes sense to hide the field.